<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayControls</class>
 <widget class="QWidget" name="PlayControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>140</y>
     <width>700</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout" stretch="1,10,0,1,0,1,1,0">
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QToolButton" name="PlayPause">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/playcontrols/images/Play.png</normaloff>:/playcontrols/images/Play.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="mediaPosition">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::add-page:horizontal {
    background: rgb(230,230, 230)
}
QSlider::sub-page:horizontal{
    background: rgb(230, 230, 230)
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="positioLabel">
      <property name="styleSheet">
       <string notr="true">color: #c5c5c5</string>
      </property>
      <property name="text">
       <string>0/0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="muteButton">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/playcontrols/images/speaker.png</normaloff>:/playcontrols/images/speaker.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="volumeControl">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"> QSlider::groove:horizontal {
     border: 1px solid #999999;
     height: 2px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
     background: #e5e5e5;
 /*    margin: 2px 0;*/
 }

QSlider::handle:horizontal{
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #DFDFDF, stop: 0.5 #7E7E7E, stop:1 #BCBCBC);
	margin: -20px 0;
    border-radius: 10px;
	border: 1px solid rgb(208, 204, 204);
	width: 40px;
    height: 40px;
}
QSlider::add-page:horizontal {
    background: rgb(230,230, 230)
}
QSlider::sub-page:horizontal{
    background: rgb(230, 230, 230)
}</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="fullScreenButtonn">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/playcontrols/images/window_fullscreen.png</normaloff>:/playcontrols/images/window_fullscreen.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="cpuUsage">
      <property name="styleSheet">
       <string notr="true"> QProgressBar {
     border: 2px solid grey;
     border-radius: 5px;
	 text-align: center;
     color: #c5c5c5;
 }

 QProgressBar::chunk {
     background-color:  #827f00;
 }</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>CPU: %p%</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="optionsButtonn">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/playcontrols/images/Options2.png</normaloff>:/playcontrols/images/Options2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
